### PR TITLE
[13.0][MIG/IMP] hr_employee_identification

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: hr_employee_identification/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+To configure this module, you need to:
+
+#. go to Human Resources / Configuration / Identification types to manage your types of identification
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. open some employee form and fill in your values in the tab `Personal Information`
```
Comment (Saran440) [inline hr_employee_identification/README.rst]:
```suggestion
#. open some employee form and fill in your values in the tab `Private Information`
```

File: hr_employee_identification/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 Therp BV <https://therp.nl>
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Employee identification",
+    "version": "13.0.1.0.0",
+    "author": "Therp BV,Onestein,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Human Resources",
+    "summary": "Record more data about employees' identity documents",
```
Comment (Saran440) [inline hr_employee_identification/__manifest__.py]:
```suggestion
    "summary": "Record more data about employee's identity documents",
```

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Comment by astirpe** [comment]
Thanks @Saran440

**Review by dsolanki-initos — APPROVED** [review]
Functional test and code review. LGTM

**Comment by dsolanki-initos** [comment]
Hi @astirpe ,
It seems that PR is ready for merge can you please ask the PSC team member and get this merge?

**Pull request merged**